include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -3397,12 +3397,11 @@ __parallel_set_union_op(__parallel_tag<_IsVector> __tag, _ExecutionPolicy&& __ex
         __par_backend::__parallel_invoke(
             __backend_tag{}, ::std::forward<_ExecutionPolicy>(__exec),
             [=] {
-                __internal::__pattern_walk2_brick(__tag, ::std::forward<_ExecutionPolicy>(__exec), __first1, __last1,
-                                                  __result, __copy_range);
+                __internal::__pattern_walk2_brick(__tag, std::move(__exec), __first1, __last1, __result, __copy_range);
```

Comment:
Probably `std::move(__exec)` will give us compile error here in some cases.
I propose simple to pass `__exec` as is.

Reply:
For experiment I merged this PR into https://github.com/uxlfoundation/oneDPL/pull/2374 - let's check will we have compile errors here or no...
Let's see the results in https://github.com/uxlfoundation/oneDPL/pull/2374/checks

Reply:
Looks this variant compiled without errors, so I agree with this change.

Reply:
Let's resolve this conversation.